Dev/src/screens/TeacherScreens/ClassTabs/ClassMainScreen.js

```diff
@@ -74,6 +75,12 @@ export class ClassMainScreen extends QcParentScreen {
     );
 
   }
+//to write implemntation of the function, updates class name
+updateTitle(newTitle){
+  this.setState({titleHasChanged: true})
+  this.setState({currentClass: {...this.state.currentClass, name: newTitle}})

```

Comment:
In this case, it may be easier to just have a "name" field defined in the state so you don't have to update the entire class object every time. Then you can just have one setState method: this.setState({ titleHasChanged: true, name: newTitle });
Then in the following TopBanners, just have title={this.state.name}